website/docs/r/monitor_cloud_account.md

```diff
@@ -15,18 +15,31 @@ Creates a Sysdig Monitor Cloud Account for monitoring cloud resources.
 ## Example Usage
 
 ```terraform
+// GCP example
 resource "sysdig_monitor_cloud_account" "sample" {
   cloud_provider = "GCP"
   integration_type = "API"
   account_id = "gcp_project_id"
+  secret_key = "sdasdasagfdkgmjsdkgmsdlkgndgsngklsdngksdlngsdsg"
+}
+
+// AWS example
```

Comment:
Can we change to:
`// AWS example with role delegation`

and add another example  with only `secret_key` and `secret_key_id`

Reply:
I didn't try with that config, could you please provide me and example to test it before I apply the changes??